components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Data Feeds"
+description = "Export Node Data from Chef Client Runs to External Data Collectors"
+date = 2020-05-05T13:19:02-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 15
++++
+
+The Data Feed service sends node data to a 3rd party service.
+This can be useful when updating Configuration Management Databases, external Security Dashboards and IT service management platforms.
```

Comment:
Those are not proper names, are they? I do not think they should be capitalized.
```suggestion
This can be useful when updating configuration management databases, external security dashboards and IT service management platforms.
```